src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/subqueries-in-nrql.mdx

```diff
@@ -35,7 +35,7 @@ During query execution, each subquery is run independently, and its result is us
 
 The [query duration limit](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#query-duration) is honored for queries with subqueries. This means that all subqueries and the outer query must complete execution within the duration limit.
 
-The maximum number of results a subquery can return is the same as the [`LIMIT MAX` value](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit): 2,000. When that limit is exceeded, it may result in incomplete results and this error message: "Subquery may have reached the maximum result member limit, which may cause an incomplete result."
+The maximum number of results a subquery can return is the same as the [`LIMIT MAX` value](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit): 5,000. When that limit is exceeded, it may result in incomplete results and this error message: "Subquery may have reached the maximum result member limit, which may cause an incomplete result."
```

Comment:
```suggestion
The maximum number of results a subquery can return is the same as the [`LIMIT MAX` value](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit): 5000. When that limit is exceeded, it may result in incomplete results and this error message: "Subquery may have reached the maximum result member limit, which may cause an incomplete result."
```